rotkehlchen/api/v1/schemas.py

```diff
@@ -975,6 +976,9 @@ class ModifiableSettingsSchema(Schema):
     )
     cost_basis_method = SerializableEnumField(enum_class=CostBasisMethod, load_default=None)
     eth_staking_taxable_after_withdrawal_enabled = fields.Boolean(load_default=None)
+    address_name_priority = fields.List(fields.String(
+        validate=webargs.validate.OneOf(choices=DEFAULT_ADDRESS_NAME_PRIORITY),
+    ), load_default=None)
```

Comment:
Why have `load_default=None` here? You can actually use the default priority as the default choice if nothing is given. This way the argument won't be optional at all.